### PR TITLE
Minor fixes

### PR DESCRIPTION
**Pull request by patrickxb**
A linter I was using found these.  Nothing urgent.

### CODE REVIEW

File: go/chat/convsource.go [hunk 1]
```diff
@@ -216,6 +216,7 @@ func (s *RemoteConversationSource) Pull(ctx context.Context, convID chat1.Conver
 		Reason:         reason,
 	}
 	boxed, err := s.ri().GetThreadRemote(ctx, rarg)
+	// XXX rl is never used
```
Comment (mmaxim) [inline go/chat/convsource.go]: Can just remove this, I just left it from the refactor of this stuff.

File: go/chat/push.go [hunk 2]
```diff
@@ -982,6 +982,7 @@ func (g *PushHandler) SetTeamRetention(ctx context.Context, m gregor.OutOfBandMe
 				if uiItem != nil {
 					convUIItems = append(convUIItems, *uiItem)
 				}
+				// XXX staleUpdates is never used
```
Comment (mmaxim) [inline go/chat/push.go]: Juts remove stakle updates

File: go/chat/convsource.go (hunk 1 above)
Comment (patrickxb) [reply go/chat/convsource.go]: done

File: go/chat/push.go (hunk 2 above)
Comment (patrickxb) [reply go/chat/push.go]: done

**Pull request merged**